tools/proto/protoc-gen-go-compat/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"path"
+	"strings"
+
+	"google.golang.org/protobuf/compiler/protogen"
+)
+
+func main() {
+	protogen.Options{}.Run(func(gen *protogen.Plugin) error {
+		for _, f := range gen.Files {
+			if !f.Generate {
+				continue
+			}
+			generateFile(gen, f)
+		}
+
+		return nil
+	})
+}
+
+// generateFile generates a _ascii.pb.go file containing gRPC service definitions.
```

Comment:
nit: the description is probably not in line with what the function actually does.

Reply:
Documented for post-migration improvements.